element_lab/project_with_study.py

```diff
@@ -0,0 +1,102 @@
+import importlib
+import inspect
+
+import datajoint as dj
+
+from element_lab import lab

```

Comment:
```suggestion
```

Reply:
Unless I am mistaken, this shouldn't be required because of the `activate` method.